admob/AdMobExample/ViewController.m

```diff
@@ -79,7 +79,7 @@ - (void)interstitialDidDismissScreen:(GADInterstitial *)interstitial {
 }
 
 - (IBAction)didTapInterstitialButton:(id)sender {
-  if ([self.interstitial isReady]) {
+  if ((self.interstitial).isReady) {
```

Comment:
The parentheses are superfluous. Xcode puts them because he doesn't know for sure if they're needed or not. It happened to me too in a real application and I went and removed them manually.